tests/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace Step\Acceptance\Administrator;
+
+class Content extends \AcceptanceTester
+{
+	/**
+	 * @When I visit articles page
+	 */
+	public function iVisitArticlesPage()
+	{
+		$I = $this;
+		$I->waitForElement(\Page\Acceptance\Administrator\com_content\Articles::$titleLocator);
+		$I->see(\Page\Acceptance\Administrator\ControlPanel::$title, \Page\Acceptance\Administrator\ControlPanel::$titleLocator);
+		$I->seeCurrentUrlEquals(\Page\Acceptance\Administrator\ControlPanel::$url);	}
```

Comment:
This looks like:

![screen shot 2016-06-10 at 12 38 49](https://cloud.githubusercontent.com/assets/1375475/15962001/68d41ae8-2f08-11e6-989a-7f90310b8658.png)